WebApp/autoreduce_webapp/autoreduce_webapp/settings.py

```diff
@@ -139,7 +139,7 @@ def get_str(section, key):
 STATIC_URL = '/static/'
 STATIC_PATH = os.path.join(BASE_DIR, 'static')
 STATICFILES_DIRS = [
-    os.path.join('static'),
+    os.path.join('BASE_DIR, static'),
```

Comment:
```suggestion
    os.path.join(BASE_DIR, 'static'),
```
Just a small typo because the ' was making this whole thing a string!